pkgpanda/cli.py

```diff
@@ -123,7 +123,13 @@ def run_checks(checks, install, repository):
 
 
 def main():
-    arguments = docopt(__doc__, version="Pkgpanda Package Manager")
+    arguments = docopt(
+        __doc__.format(
+            default_config_dir=constants.config_dir,
+            default_root=constants.install_root,
+            default_repository=constants.repository_base,
+        ),
```

Comment:
Prior discussion here: https://github.com/dcos/dcos/pull/91#discussion_r71184369

I went ahead and removed it since I didn't want to get into a discussion over what the version string ought to be in this PR.